CHANGES.md

```diff
@@ -59,6 +59,7 @@ be compatible with this version.
     `List<IReadOnlyList<string>>?` from `List<List<string>>?`.  [[#3274]]
  -  (Libplanet.Explorer) Changed the type for `TxResult.ActionsLogList` to
     `List<IReadOnlyList<string>>?` from `List<List<string>>?`.  [[#3274]]
+ -  (Libplanet.Explorer) Added `BlockType.PreEvaluationHash` field. [[#3281]]
```

Comment:
```suggestion
 -  (Libplanet.Explorer) Added `BlockType.PreEvaluationHash` field. [[#3280], [#3281]]
```
Must mention the resolving issue (or related thereof) as well.

Also, you must add the following to the end of the changelog for this version:
```md
[#3280]: https://github.com/planetarium/libplanet/issues/3280
[#3281]: https://github.com/planetarium/libplanet/pull/3281
````

Reply:
I just thought it's link doesn't work because it takes some time to sync. Thank you for let me know. 👍